### PR TITLE
Default tag should contains all latest apiVersion swagger file 

### PR DESCRIPTION
**Pull request by ruowan**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -309,6 +330,167 @@ const findTheNearestReadme = async (rootDir: string, swaggerPath: string): Promi
...
 
+export type PathTable = Map<string, { apiVersion: string; swaggerFile: string }>
+
+export const validateRPMustContainAllLatestApiVersionSwagger = (dir: string): it.IterableEx<err.Error> =>
+  it.iterable<err.Error>(function*() {
+    const readmePattern = path.join(dir, '**/readme.md')
+    const readmes = glob.sync(readmePattern, { nodir: true })
+
+    for (const readme of readmes) {
+      console.log(readme)
```
Comment (jianyexi) [inline src/index.ts:341]: is this debug info ?

File: src/index.ts [hunk 2]
```diff
@@ -309,6 +330,167 @@ const findTheNearestReadme = async (rootDir: string, swaggerPath: string): Promi
...
+  const pathTable = new Map<string, { apiVersion: string; swaggerFile: string }>()
+  // tslint:disable-next-line: no-shadowed-variable
+  for (const it of allPaths) {
+    pathTable.set(it, { apiVersion, swaggerFile })
+  }
+  return pathTable
+}
+
+export const getAllPathFromSwagger = (swagger: any) => {
+  const apiPath = '$.paths.*~'
```
Comment (jianyexi) [inline src/index.ts:466]: needs to consider 'x-ms-paths' as well

**Review by jianyexi — APPROVED** [review]

**Comment by chgennar** [comment]
@ruowan - i have the MISSING_APIS_IN_DEFAULT_TAG error showing up in a PR.

image

In my particular case, it looks like a change in casing is causing a discrepancy between an earlier and new version of the same API. Should this MISSING_APIS_IN_DEFAULT_TAG use a case insensitive comparison? Or possibly should it rely on the operation ids instead of the paths?

**Comment by chgennar** [comment]
> Thanks for your feedback. This errors use case sensitive comparison for paths and normalize path by ignore parameter. Operation id is more flexible than path.
> 
> I think I need to update this rule to use case insensitive comparison.
> 
> Could you share the PR link?

This is the check that is failing on my PR. I think it's caused by a change in casing between two spec versions.
https://github.com/Azure/azure-rest-api-specs/pull/17439/checks?check_run_id=5378743491

**Comment by mc3784** [comment]
Hi @ruowan 
the avocado check on my pr is failing with this errors: MISSING_APIS_IN_DEFAULT_TAG and NOT_LATEST_API_VERSION_IN_DEFAULT_TAG
image

I am not able to understand how to fix them, and would really appreciate some help. Here the link to the PR:
https://github.com/Azure/azure-rest-api-specs-pr/pull/6412/checks?check_run_id=5409374092

**Pull request merged**